code/ai/aigoals.cpp

```diff
@@ -1413,21 +1413,18 @@ int ai_mission_goal_achievable( int objnum, ai_goal *aigp )
 	{
 		ship *shipp = &Ships[objp->instance];
 
-		// always valid if has subspace drive
-		if (!(shipp->flags[Ship::Ship_Flags::No_subspace_drive]))
+		// always valid if has working subspace drive and not disabled
+		if ( ship_can_warp_full_check(shipp) )
 			return AI_GOAL_ACHIEVABLE;
 
 		// if no subspace drive, only valid if our mothership is present
```

Comment:
tweak this comment because the if() check immediately above it now checks additional things

Reply:
You tweaked the wrong comment...